src/class/cdc/cdc_host.c

```diff
@@ -813,7 +818,6 @@ static bool acm_set_baudrate(cdch_interface_t* p_cdc, tuh_xfer_cb_t complete_cb,
   p_cdc->requested_line_coding.stop_bits = p_cdc->line_coding.stop_bits;
   p_cdc->requested_line_coding.parity    = p_cdc->line_coding.parity;
   p_cdc->requested_line_coding.data_bits = p_cdc->line_coding.data_bits;
-  TU_VERIFY(p_cdc->acm_capability.support_line_request);
```

Comment:
this was missed in acm_set_data_format().
moved them to common called acm_set_line_coding()